docs/docs/guides/token_expiration.mdx

```diff
@@ -0,0 +1,103 @@
+---
+id: token_expiration
```

Comment:
All other pages use hyphens so let's keep that convention. Please rename the file to `token-expiration.mdx`

```suggestion
id: token-expiration
```